Details on the sequencer fee vault (#858)
ethereum-optimism/docs: pages/stack/transactions/fees.mdx

```

The new cost function takes into account the compression ratio, so chain operators will need to adjust their `baseFeeScalar` and `blobFeeScalar`  to account for any previous compression ratios that they encountered on their chains. Chain operators can use the [Fjord fee parameter calculator](https://docs.google.com/spreadsheets/d/1V3CWpeUzXv5Iopw8lBSS8tWoSzyR4PDDwV9cu2kKOrs/edit#gid=186414307) to get a better estimate of scalar values to use for your chain.

## Sequencer Fee Vault

> 
> The **Sequencer Fee Vault** is responsible for collecting and holding transaction fees paid to the sequencer during the processing and production of blocks on OP Mainnet. These fees are used to cover the cost of posting transaction data to Ethereum Layer 1 (L1), ensuring that the network remains sustainable and that data availability is maintained.

Reviewer: ```suggestion

The Sequencer Fee Vault collects and holds transaction fees paid to the sequencer during block production on OP Mainnet. These fees cover the cost of posting transaction data to L1, ensuring network sustainability and data availability.

```

Reviewer: @richardgreg this?

### Fee Collection and Distribution

> 
> *   **Purpose**: The fees collected by the sequencer are deposited into the Sequencer Fee Vault. These fees are used to reimburse the sequencer for the gas costs incurred when submitting transaction batches to L1.
> *   **Vault Address**: The Sequencer Fee Vault is predeployed at the address `0x4200000000000000000000000000000000000011` on the OP Mainnet.
> *   **Fee Usage**: The fees stored in the vault are eventually transferred to the designated recipient address, which could be a treasury or another contract responsible for distributing the funds.

Reviewer: ```suggestion
*   **Purpose**: The sequencer deposits collected fees into the Sequencer Fee Vault. These fees reimburse the sequencer for gas costs when submitting transaction batches to L1.
*   **Vault Address**: The Sequencer Fee Vault is predeployed at the address `0x4200000000000000000000000000000000000011` on the OP Mainnet.
*   **Fee Usage**: Stored fees are eventually transferred to a designated recipient address (e.g., a treasury or distribution contract).

```

Reviewer: @richardgreg this?

### How it Works

1.  **Fee Collection**: During the processing of transactions, the sequencer collects fees from users as part of their transaction costs. These fees are primarily used to cover the gas expenses of posting transaction data to Ethereum L1.
2.  **Storage**: Collected fees are deposited into the Sequencer Fee Vault contract.

> 3.  **Distribution**: The fees are later distributed to the appropriate recipient, typically to cover operational costs like gas fees for data availability on L1.
> 
> This system ensures that fees are managed effectively, helping to keep the Optimism network secure and operational.

Reviewer: ```suggestion
3.  **Distribution**: The fees are later distributed to the appropriate recipient, typically covering operational costs like L1 gas fees for data availability.

This system ensures effective fee management, helping maintain the security and operation of the Optimism network.

```

Reviewer: @richardgreg this?
